### ISSUE TITLE
suggestion: keyboard shortcut for logcat and dmesg

### ISSUE THREAD

**Issue by Loader009** [tdm/android_device_fxtec_pro1#49]
For debugging purposes I suggest a keyboard shortcut to save logcat and dmesg.
power+camera?

This would be very helpful for the display issue, since I can do a force reset right after it happens.

**Comment by tdm** [comment]
Perhaps this?

https://developer.android.com/studio/debug/bug-report

**Comment by Loader009** [comment]
Yes, this works, logcat and dmesg and pretty much everything else is being included within a bugreport.
With `adb shell ls /bugreports` you can list all bugreports and with `adb pull /bugreports/bugreport...` you can get it.

But before anyone posts those logfiles, we might better provide some script to strip private information out of it? (Which is kinda hard for various data.)
